include/options/abstract-option.php

```diff
@@ -240,4 +221,13 @@ protected function sanitize( $value, Options $options ) { // phpcs:ignore Variab
 	 * @phpstan-return array{type: SchemaType}
 	 */
 	abstract protected function get_specific_schema(): array;
+
+	/**
+	 * Returns the description used in the JSON schema.
+	 *
+	 * @since 3.7
+	 *
+	 * @return string
+	 */
+	abstract protected function get_description(): string;
```

Comment:
```suggestion
	abstract protected function description(): string;
```
Same as for `default()`.

Reply:
I prefer to keep as it is.